sciserver_catalogs/environment.yml

```diff
@@ -37,8 +36,10 @@ dependencies:
   - cf_xarray
   - oceanspy==0.2.0
```

Comment:
This might be the problem in #221  You want to either pip install the main version from git or make a new release.

Reply:
I think that might be it. The `AttributeError` shows snippet of an older version of `oceanspy`

![Screen Shot 2022-02-16 at 6 48 43 PM](https://user-images.githubusercontent.com/8241481/154376801-fc64ddb2-920a-485a-ae34-0bb6d92b8314.png)